### PR TITLE
Task/users filter

### PR DESCRIPTION
**Pull request by JuanGro**
#### What's this PR do?
To implement users filter, pagination and change feedback style.

#### Where should the reviewer start?
With the filter.

#### How should this be manually tested?
Looking for users by first_name, last_name and username.

#### Any background context you want to provide?
No.

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: src/app/admin/users/users.component.ts [hunk 1]
```diff
@@ -141,6 +147,29 @@ export class UsersComponent implements OnInit {
...
 
   /**
+  * Shows the user list that the user is requesting in the filter.
+  * Params:
+  *   - value: String from search form.
+  **/
+  public getValueSearch(value: string) {
+    this.valueSearch = value;
+    this.userList = [];
+    this.completeUserList.sort();
```
Comment (EduardoVaca) [inline src/app/admin/users/users.component.ts]: Sort this List the first time is created. There is not need to sort the completeUserList every time something is searched

File: src/app/layouts/full-layout.component.html [hunk 2]
```diff
@@ -304,7 +304,7 @@
                 <li class="nav-item nav-dropdown" routerLinkActive="open">
                     <a class="nav-link" routerLinkActive="active" [routerLink]="['/admin/completed-designs-list']"><i class="glyphicons glyphicons-crown"></i> Diseños terminados</a>
                 </li>
-                <li *ngIf="admin == true" class="nav-item nav-dropdown" routerLinkActive="open">
+                <li class="nav-item nav-dropdown" routerLinkActive="open">
```
Comment (EduardoVaca) [inline src/app/layouts/full-layout.component.html]: Please put 
`*ngIf="currentUser.hasRole([ roleEnum.SUPER_USUARIO ])"`

Comment (EduardoVaca) [inline src/app/layouts/full-layout.component.html]: To add RBAC to this module of users

File: src/app/admin/users/users.component.ts (hunk 1 above)
Comment (JuanGro) [inline src/app/admin/users/users.component.ts]: Done

File: src/app/layouts/full-layout.component.html (hunk 2 above)
Comment (JuanGro) [inline src/app/layouts/full-layout.component.html]: Done

**Pull request merged**